src/utils/actions.js

```diff
@@ -1,5 +1,80 @@
+import React from "react";
+import { Avatar } from "react-native-elements";
+import { View, Text } from "react-native";
+
+// Created this functional component to fix current Avatar flickering issue on ios
+// Should be fixed in the next React Native Elements Update
+const userAvatar = ({ item }) => {
+	return (
+		<Avatar
+			size = "large"
+			rounded
+			source = {{ uri: item.picture }}
+		/>
+	);
+};
+export const StableAvatar = React.memo(userAvatar);
+
 export const createActiontypes = actionTypes => actionTypes
```

Comment:
Following camelCase, can types be capitalized? `createActiontypes` -> `createActionTypes`

Reply:
Also, add parenthesis around the parameter (even if not needed) to differentiate parts of the function.